extension/chrome/elements/compose-modules/compose-err-module.ts

```diff
@@ -160,6 +161,15 @@ export class ComposeErrModule extends ViewModule<ComposeView> {
   };
 
   public throwIfEncryptionPasswordInvalidOrDisabled = async ({ subject, pwd }: { subject: string; pwd?: string }) => {
+    const disallowedPasswordMessageTerms = this.view.clientConfiguration.getDisallowPasswordMessagesForTerms();
+    const disallowedPasswordMessageErrorText = this.view.clientConfiguration.getDisallowPasswordMessagesErrorText();
+    if (
+      disallowedPasswordMessageErrorText &&
+      disallowedPasswordMessageTerms &&
+      subject.split(' ').some(term => disallowedPasswordMessageTerms.includes(term))
```

Comment:
I missed that here we search only for separate words, as it's possible to have multiple words in a single term (like `internal plan`), and it'll also fail for cases when word is surrounded by brackets like `[INTERNAL]`.
So let's search for terms in full subject string.

Reply:
We've also got a bit more details from customer (https://github.com/FlowCrypt/flowcrypt-android/issues/2905#issuecomment-2529418537):
- search should be case-insensitive
- possible string values are `['ABDU', '[Classification: Data Control: Internal Data Control]']`